template_feed/Microsoft.DotNet.Common.ItemTemplates/content/BuildProps/.template.config/template.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": [
+    "MSBuild",
+    "props"
+  ],
+  "name": "MSBuild Directory.Build.props file",
+  "generatorVersions": "[1.0.0.0-*)",
+  "description": "An empty Directory.Build.props file which can be used to specify MSBuild properties for an entire folder",
+  "tags": {
+    "type": "item"
+  },
+  "identity": "Microsoft.Standard.QuickStarts.DirectoryProps",
+  "groupIdentity": "MSBuildDirectoryPropsFile",
+  "precedence": 0,
+  "shortName": "buildprops",
+  "symbols": {
+    "inherit": {
```

Comment:
One thing to keep in mind is that Visual Studio does not show options currently for item template symbols.

Reply:
I think we talked about that on the last iteration of these changes and decided that was Ok for now. Part of the value of the template is educational (letting people know that this is a thing that can be done), and even without the flag the generated file is correct, so I think there's still value here.